apps/api-gateway/src/authz/guards/ecosystem-roles.guard.ts

```diff
@@ -54,19 +50,17 @@ export class EcosystemRolesGuard implements CanActivate {
       const ecosystemOrgData = await this.ecosystemService.fetchEcosystemOrg(ecosystemId, orgId);
 
       if (!ecosystemOrgData) {
-        throw new HttpException('Organization does not match', HttpStatus.FORBIDDEN);
+        throw new ForbiddenException('Organization does not match');
```

Comment:
Please add these messages in the common file

Reply:
Resolved.